akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -283,17 +283,30 @@ Next the actual SSL/TLS parameters have to be configured:
 akka.remote.artery {
   transport = tls-tcp
 
-  ssl.config-ssl-engine {
-    key-store = "/example/path/to/mykeystore.jks"
-    trust-store = "/example/path/to/mytruststore.jks"
-
-    key-store-password = ${SSL_KEY_STORE_PASSWORD}
-    key-password = ${SSL_KEY_PASSWORD}
-    trust-store-password = ${SSL_TRUST_STORE_PASSWORD}
-
-    protocol = "TLSv1.2"
-
-    enabled-algorithms = [TLS_DHE_RSA_WITH_AES_128_GCM_SHA256]
+  ssl {
+    ssl-engine-provider = akka.remote.artery.tcp.ConfigSSLEngineProvider
+
+    config-ssl-engine {
+      key-store = "/example/path/to/mykeystore.jks"
+      trust-store = "/example/path/to/mytruststore.jks"
+  
+      key-store-password = ${SSL_KEY_STORE_PASSWORD}
+      key-password = ${SSL_KEY_PASSWORD}
+      trust-store-password = ${SSL_TRUST_STORE_PASSWORD}
+  
+      protocol = "TLSv1.2"
+  
+      enabled-algorithms = [TLS_DHE_RSA_WITH_AES_128_GCM_SHA256]
```

Comment:
should the example include `"TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"` as in https://github.com/akka/akka/pull/29049 ?